packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx

```diff
@@ -288,7 +288,20 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
   }
 
   const limit = Number(props.sddLimit.max) / SDD_LIMIT_FACTOR
-
+  const isErc20 =
+    props.payment &&
+    (props.payment.coin === 'PAX' ||
+      props.payment.coin === 'USDT' ||
+      props.payment.coin === 'WDGLD')
+  const isSufficientEthForErc20 =
+    props.payment &&
+    (props.payment.coin === 'PAX' ||
+      props.payment.coin === 'USDT' ||
+      props.payment.coin === 'WDGLD') &&
```

Comment:
shouldnt need to repeat this check again. I think you can just use the flag set above?

Reply:
The check needs to be repeated, because `isSufficientEthForErc20` doesn't exist on the BtcPaymentValue, so in the declaration, it needs to be v. explicit that it's an Eth payment value. I tried removing it and received a TS error.